superset/config.py

```diff
@@ -380,9 +380,16 @@ def _try_json_readsha(  # pylint: disable=unused-argument
 # Setup image size default is (300, 200, True)
 # IMG_SIZE = (300, 200, True)
 
+# Default timeout for cache (1 day)
```

Comment:
Do you think it's worthwhile also adding a comment to say it applies to all caches unless overridden at the cache level?

Reply:
Done.